lib/github-pages/configuration.rb

```diff
@@ -19,6 +19,7 @@ class Configuration
       "kramdown" => {
         "input"     => "GFM",
         "hard_wrap" => false,
+        "gfm_quirks" => "paragraph_end",
```

Comment:
I can't think of any, other than perhaps wanting to change other flags down the line (which would be in that same string).